### ISSUE TITLE
400 BAD REQUEST when using :addReaction()

### ISSUE THREAD

**Issue by Sukadia** [SinisterRectus/Discordia#224]
I'm trying to get the bot to react to it's own message right after its sent so it can check when a user adds to it. However, no matter which reaction I try to add, it throws me this error:

`2020-01-23 20:05:29 | [ERROR]   | 400 - BAD REQUEST : PUT https://discordapp.com/api/v7/channels/670116945701634063/messages/670116946188173314/reactions/�/@me`

```lua
client:on("ready", function()
    print("Server Setup")
    server = client:getGuild("669338665956409374")
    local queue = server:createTextChannel("Queue")
    local message = queue:send("React to confirm your submission into The Queue.")
    message:addReaction("👍")
    client:on("reactionAdd",function(reaction,id)
        print(reaction)
    end)
    
    
    print("Server has been setup")
end)
```

In discord, it properly creates the channel and sends the message, but the reaction never comes through and the bot continues to run after it errors.

Any work-arounds or ways to fix this issue? Thanks in advance

**Comment by SinisterRectus** [comment]
1. Check your version. This bug was fixed in 2.8.0.
2. The event handler for "reactionAdd" should be registered before the request is made.

**Comment by Sukadia** [comment]
Yep, the version was the issue, was running on 2.6.0 somehow. I really appreciate the fast response, it was quite frustrating, thank you!
